pkl-executor/src/test/kotlin/org/pkl/executor/EmbeddedExecutorTest.kt

```diff
@@ -415,6 +420,60 @@ class EmbeddedExecutorTest {
       .doesNotContain(tempDir.toString())
   }
 
+  @ParameterizedTest
+  @MethodSource("getAllTestExecutors")
+  fun `evaluate a module whose project evaluation fails`(
+    executor: TestExecutor,
+    @TempDir tempDir: Path
+  ) {
+    tempDir
+      .resolve("PklProject")
+      .toFile()
+      .writeText(
+        """
+      amends "pkl:Project"
+      dependencies {
+        ["myDep"] = import("../nonexistent/PklProject")
+      }
+    """
+          .trimIndent()
+      )
+
+    val pklFile = tempDir.resolve("test.pkl")
+    pklFile
+      .toFile()
+      .writeText(
+        """
+      @ModuleInfo { minPklVersion = "0.11.0" }
+      module test
+
+      foo = "bar"
+    """
+          .trimIndent()
+      )
+
+    val e =
+      assertThrows<ExecutorException> {
+        executor.evaluatePath(pklFile) {
+          allowedModules("file:")
+          allowedResources("prop:")
+          projectDir(tempDir)
+          rootDir(tempDir)
+        }
+      }
+
+    assertThat(e.message)
+      .contains("Cannot find module")
+      // ensure module file paths are relativized
+      .contains("/nonexistent/PklProject")
+
+    // legacy distribution does not handle these errors with the correct stack frame transformer
+    // only assert on this for newer distributions
+    if (!executor.toString().contains("Distribution1")) {
+      assertThat(e.message).doesNotContain(tempDir.toString())
+    }
```

Comment:
Not so sure what this assertion is about, but it's failing on my machine

Reply:
You might see this if you haven't built  since switching to this branch since EmbeddedExecutorTest loads the built `pkl-config-java/build/libs/pkl-config-java-all-SNAPSHOT.jar`. The tests error if this file is not present, but not if it's out of date.

Reply:
Ah, gotcha!